components/TaskCard.vue

```diff
@@ -164,6 +176,14 @@ export default {
 
       this.showEditIcon = true;
       this.loading = false;
+      this.alert();
+    },
+    alert() {
+      swal(this.$t('alert.message.success'), {
+        buttons: false,
+        className: 'success',
+        timer: 3000,
+      });
```

Comment:
Let's move it into `utils.js` and reuse it.

Reply:
resolved